.github/workflows/test-changed-firestore.yml

```diff
@@ -0,0 +1,31 @@
+name: Test Firestore
+
+on: pull_request
+
+jobs:
+  test:
+    name: Test Firestore
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@master
+      with:
+        # This makes Actions fetch all Git history so run-changed script can diff properly.
+        fetch-depth: 0
+    - name: Set up Node (10)
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+    - name: install Chrome stable
+      run: |
+        sudo apt-get update
+        sudo apt-get install google-chrome-stable
+    - name: Test setup and yarn install
+      run: |
+        cp config/ci.config.json config/project.json
+        yarn
+    - name: yarn build
+      run: yarn build
+    - name: Run tests if firestore or its dependenceies has changed
+      run: xvfb-run yarn test:changed:firestore
```

Comment:
Does firestore require xvfb-run? As long as it doesn't hurt I guess.

Reply:
I think they are only needed for tests that use headful browsers. Removed unnecessary xvfb-runs.